README.md

```diff
@@ -164,6 +164,23 @@ You can add additional parameters (like `--version=1.0.0`) at the end of the com
 List of supported arguments is compatible with
 [`helm show values`](https://helm.sh/docs/helm/helm_show_values/).
 
+### Jsonnet prometheus-operator configuration
+
+`get-jsonnet` is a command which generates `remoteWrite` configuration for the kube prometheus.
+
+```
+```
+ kubectl run template-dependency \
+  -it --quiet --rm \
+  --restart=Never -n sumologic \
+  --image sumologic/kubernetes-tools \
+  -- get-libjsonnet > kube-prometheus-sumo-logic-mixin.libsonnet
+```
```

Comment:
Lol the suggestions breaks around backticks but there's one superfluous line with 3 backticks (can't even type them here 🤷🏻 )